R/Model-class.R

```diff
@@ -207,29 +255,27 @@ LogisticNormal <- function(mean,
     alpha1 <- theta[2]
   }
 
+  params <- ModelNormal(mean = mean, cov = cov, ref_dose = ref_dose)
   .LogisticNormal(
     datanames = c("nObs", "y", "x"),
     datamodel = jags_model_data,
     priormodel = jags_model_prior,
     modelspecs = function() {
-      list(refDose = refDose, prec = prec, mean = mean)
+      list(ref_dose = params@ref_dose, mean = params@mean, prec = params@prec)
     },
     init = function() {
       list(theta = c(0, 1))
     },
     sample = c("alpha0", "alpha1"),
     dose = function(prob, alpha0, alpha1) {
       stand_log_dose <- (logit(prob) - alpha0) / alpha1
-      exp(stand_log_dose) * refDose
+      exp(stand_log_dose) * ref_dose
     },
     prob = function(dose, alpha0, alpha1) {
-      stand_log_dose <- log(dose / refDose)
+      stand_log_dose <- log(dose / ref_dose)
       plogis(alpha0 + alpha1 * stand_log_dose)
     },
-    mean = mean,
-    cov = cov,
-    prec = prec,
-    refDose = refDose
+    params
```

Comment:
I would start with this on top of the construction call.
also maybe rename to `start` or `object` ?

Reply:
I though it would be if the order of arguments corresponds to the order ot classes inheritance.

Reply:
order: it is easier to read if it goes on top - then as a reader you see it right away. also inheritance wise is that order - first the parent, then the specific children slots.
name I don't have strong opinion, just for me I think of it as a "start object"

Reply:
Sure. I have just submitted the new version with the new order. Please let me know if it is ok. Tx!